### PR TITLE
[release-2.7.x] fix(promtail): Make `cri` tags streams aware. (#8497)

### PR DESCRIPTION
**Pull request by DylanGuedes**
(cherry picked from commit 43a21eac0379d7f9320cd94f2d27a3a46bd19d3f)

### CODE REVIEW

**Review by chaudum — COMMENTED** [review]
This PR does not have a changelog entry. The initial functionality has already been released, right?

**Comment by DylanGuedes** [comment]
> This PR does not have a changelog entry. The initial functionality has already been released, right?

Good catch, thank you. Yeah, the initial functionality was already there, so I only added an entry mentioning the fix.

File: CHANGELOG.md [hunk 1]
```diff
@@ -44,6 +44,7 @@
 
 ##### Fixes
 
+* [8497](https://github.com/grafana/loki/pull/8497) **kavirajk**: Fix `cry` tags treating different streams as the same
```
Comment (MichelHollands) [inline CHANGELOG.md:47]:
```suggestion
* [8497](https://github.com/grafana/loki/pull/8497) **kavirajk**: Fix `cri` tags treating different streams as the same
```

**Review by MichelHollands — APPROVED** [review]
One small typo fix, fine otherwise

**Pull request merged**